gometa_test.go

```diff
@@ -84,3 +84,11 @@ func TestGometa(t *testing.T) {
 		}
 	}
 }
+
+func TestInternalFolderInPath(t *testing.T) {
+	rec := httptest.NewRecorder()
+	err := gometa(rec, record{}, "example.com", "test/internal")
```

Comment:
The test would still work for catching whether internal is allowed or not, but for correctness `path` should be an absolute path, as it will be appended to a host (so `/test/internal` would be better)